common/defs/ethereum/networks.json

```diff
@@ -367,5 +367,25 @@
     "rskip60": false,
     "url": "https://pirl.io",
     "blockbook": []
+  },
+{
+    "chain": "volta",
+    "chain_id": 73799,
+    "slip44": 44,
+    "shortcut": "VT",
+    "name": "Volta",
+    "rskip60": false,
+    "url": "https://www.energyweb.org/",
+    "blockbook": []
+  },
+  {
+    "chain": "energyweb",
```

Comment:
just to confirm: this name should match what [ethereum-lists/tokens](https://github.com/ethereum-lists/tokens) uses to identify tokens on your network. there are currently none, but please check that this is the symbol you want to use

Reply:
yes... just prepared those tokens / uploading right away

Reply:
Energyweb chain -> would that be ok since it has a %20 space in it? Or do I just use Energyweb?

Reply:
you can use `energyweb`, but that [doesn't really match](https://github.com/ethereum-lists/tokens/tree/master/tokens) what they're using currently.

i'd consider `ewc` maybe?